CHANGES.md

```diff
@@ -10,6 +10,7 @@ Features
 
 Bug Fixes
 ---------
+* [#1087](https://github.com/java-native-access/jna/pull/1087): Fix wrong calls to Structure#toArray with zero sized arrays - [@matthiasblaesing](https://github.com/matthiasblaesing).
```

Comment:
Should this be 1088?  (Yay, I found something to comment on. I did look over the PR and otherwise think it looks good.)

Reply:
If there is an issue open for a changeset, I'm referencing that. If there is no issue, then it is the PR that brought the change in. In this case the original PR is #1087 and replaces an issue.

Thank you for your review!